src/get-colors.js

```diff
@@ -1,20 +1,39 @@
-import sketch from 'sketch'
-
-export default function() {
-  const doc = sketch.getSelectedDocument()
-  const colors = doc.swatches
-
-  // Get color names from variables
-  const colorsNames = colors.map(color => {
-    return color.name.toLowerCase()
-  })
-
-  console.log(colorsNames)
-  
-  // If color variables exist, show an alert with the color names
-  if (colors.length !== 0) {
-    sketch.UI.alert('Color Variables', `${colorsNames}`)
-  } else {
-    sketch.UI.alert('No Colors', 'There is no colors!')
+import sketch from "sketch";
+import values from "./lib/values";
+
+export default () => {
+  const doc = sketch.getSelectedDocument();
+  const colors = doc.swatches;
+
+  if (!colors.length) {
+    sketch.UI.alert("No Colors", "There is no colors!");
+    return;
   }
-}
\ No newline at end of file
+
+  const formattedColors = colors.reduce(
+    (prev, color) => ({
+      ...prev,
+      [color.name.replace("Colors/", "").toLowerCase()]: color.color,
+    }),
+    {}
+  );
+
+  let type = "JSON";
+  let savePanel = NSSavePanel.savePanel();
+
+  savePanel.setNameFieldStringValue("colors." + values[type].filetype);
```

Comment:
El file de values solo se usa par traer el filetype. Esto quedaría más limpio:

```suggestion
  let type = "json";
  savePanel.setNameFieldStringValue("colors." + type);
```

Y podríamos eliminar el archivo de values.